modules/core/src/main/java/org/apache/ignite/internal/processors/query/QuerySchemaPatch.java

```diff
@@ -33,24 +32,35 @@
  */
 public class QuerySchemaPatch {
     /** Message which described conflicts during creating this patch. */
-    private String conflictsMessage;
+    private final String conflictsMessage;
```

Comment:
Why this change? Let's revert it.

Reply:
No, I disagree. Here obviously final modificator is forgotten. This field is semantically final, and QuerySchemaPatch is immutable object.